include/Core/Base/Mixin/Fetcher.php

```diff
@@ -22,6 +22,17 @@ protected function get_post_types() {
 		return bd_get_post_types();
 	}
 
+	/**
+	 * Get the list of taxompmies registered in WordPress.
+	 *
+	 * @return A list of taxonomy names
+	 */
+	protected function get_taxonomies() {
```

Comment:
If you are only going to be calling `get_taxonomies` function, then there is no need for this function.

Remove it and use `get_taxonomies` directly.